system.json

```diff
@@ -0,0 +1,203 @@
+{
+    "name": "docker_test",
+    "components": [
+        {
+            "name": "feeder",
+            "type": "LocalFeeder",
+			"host": "10.5.0.3",
+			"port": 5678,
```

Comment:
This configuration for the host and port is only used by the multi-container build for a docker compose right?